arelle/FileSource.py

```diff
@@ -4,13 +4,24 @@
 @author: Mark V Systems Limited
 (c) Copyright 2010 Mark V Systems Limited, All rights reserved.
 '''
+from __future__ import annotations
```

Comment:
Arbitrary line for discussion. Tests fail locally due to a circular import, but I believe that will be resolved when the ModelValue PR is merged.